api/src/pcapi/routes/backoffice/offerers/validation_blueprint.py

```diff
@@ -507,6 +507,19 @@ def _get_serialized_user_offerer_last_comment(
     return None
 
 
+def _display_users_offerers(user_offerers_ids: list[int]) -> utils.BackofficeResponse:
+    if user_offerers_ids:
+        items = validation_repository.list_users_offerers_to_be_validated(
+            user_offerers_ids=user_offerers_ids,
+        ).all()
+    else:
+        items = []
```

Comment:
Petit détail, c'est plus léger ainsi.
```suggestion
def _display_users_offerers(user_offerers_ids: list[int]) -> utils.BackofficeResponse:
    items = []
    if user_offerers_ids:
        items = validation_repository.list_users_offerers_to_be_validated(
            user_offerers_ids=user_offerers_ids,
        ).all()
```